package.json

```diff
@@ -18,11 +20,11 @@
     "@inquirer/confirm": "^2.0.10",
     "@inquirer/select": "^1.2.9",
     "@mangata-finance/sdk": "^1.20.1",
-    "@oak-network/adapter": "^0.1.3",
+    "@oak-network/adapter": "0.1.0-rc.21",
     "@oak-network/api-augment": "^2.0.7",
-    "@oak-network/config": "^0.1.2",
-    "@oak-network/sdk": "^0.1.3",
-    "@oak-network/sdk-types": "^0.1.3",
+    "@oak-network/config": "0.1.0-rc.21",
+    "@oak-network/sdk": "0.1.0-rc.21",
+    "@oak-network/sdk-types": "0.1.0-rc.21",
```

Comment:
Please completely get rid of this `rc.21` format. The number plus string format is hard for version comparison. Use the Github Action to publish a dev version!

Reply:
Fixed